packages/web-tracing/src/instrumentation.ts

```diff
@@ -29,7 +29,7 @@ export class TracingInstrumentation extends BaseInstrumentation {
   }
 
   initialize(): void {
-    const config = this.faro.config;
+    const config = faro.config;
```

Comment:
This change introdcues a problem if you have multiple instances of Faro, something we haven't fully solved yet.. 
Perhaps we should simply not allow multiple instances for simplicity sake?